Idref.js

```diff
@@ -0,0 +1,83 @@
+{
+	"translatorID": "271ee1a5-da86-465b-b3a5-eafe7bd3c156",
+	"label": "Idref",
+	"creator": "Sylvain Machefert",
+	"target": "^https?://www\\.idref\\.fr/.*",
```

Comment:
no need for the `.*` -- the target needs to match against the URL, not the other way around.